test_rest/mocks/v3/snapshot.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
copied from [the spec](https://github.com/polygon-io/go-apps-snapshot-consolidator/blob/master/_spec_/index.yml)